shipyard/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "3ea4d492-9d1c-4dda-bb8a-e571eca3ce7d",
+  "app_id": "shipyard",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send Shipyard logs to Datadog dashboard",
+    "title": "Shipyard",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Datadog connection settings in your Shipyard dashboard",
+        "image_url": "images/datadog-input.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Shipyard logo in teal",
+        "image_url": "images/shipyard-teal.png"
+      }
```

Comment:
```suggestion
```
The logo does not need to be added here either.  However, please add the `sample_dashboard.png` file.